src/implementations/vanilla/src/basics/text-link/tests/gemini-text-link.html

```diff
@@ -0,0 +1,83 @@
+<html>
+    <head>
+        <link rel="stylesheet" href="../../../../dist/hig.css">
+        <script type="text/javascript" src="../../../../dist/hig.js"></script>
+        <style>
+            .test-group {
+                padding: 40px;
+                margin: 0;
+                display: flex;
+                justify-content: center;
+                align-items: center;
+            }
+        </style>
+    </head>
+    <body>
+        <div class='test-group tests-text-link-defaults'></div>
+        <div class='test-group tests-text-link-with-href'></div>
+        <div class='test-group tests-text-link-secondary'></div>
+        <div class='test-group tests-text-link-secondary-with-href'></div>
+        <div class='test-group tests-text-link-with-onclick'></div>
+        <div class='test-group tests-text-link-with-onclick-and-href'></div>
+        <script>
+            // ----------------------------
+            // DEFAULT TEXTLINK
+            // ----------------------------
+
+            var textLinkWithDefaults = new Hig.TextLink();
+            textLinkWithDefaults.mount('.tests-text-link-defaults');
+
+            // ----------------------------
+            // TEXTLINK WITH LINK
+            // ----------------------------
+
+            var textLink = new Hig.TextLink({
+                href: 'https://github.com/Autodesk/hig',
+                text: 'This is a TextLink to HIG repository url'
+            });
+            textLink.mount('.tests-text-link-with-href');
+
+            // ----------------------------
+            // SECONDARY TEXTLINK
+            // ----------------------------
+
+            var secondaryTextLink = new Hig.TextLink({
+                text: 'This is a secondary TextLink',
```

Comment:
is this missing the "href" default attribute on purpose?

Reply:
it is.  trying to capture all of the possible states.  missing href will happen in practice when there's an onclick event

Reply:
oh ok cool.